.github/workflows/codecov.yml

```diff
@@ -2,7 +2,7 @@ on:
   pull_request:
   push:
     branches:
-      - main
+      - master
```

Comment:
Aaaah, yes, this why in some of the repos is not working as expected. There is kind of a mess depending on when/how the repo was created :(